controllers/clustergroupupgrade_controller.go

```diff
@@ -101,9 +102,13 @@ func (r *ClusterGroupUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.
 			allManagedPoliciesExist, managedPoliciesMissing, managedPoliciesPresent := r.doManagedPoliciesExist(ctx, clusterGroupUpgrade)
 			if allManagedPoliciesExist == true {
 				// If all the managedPolicies exist, continue with building the upgrade batches.
-				r.buildRemediationPlan(ctx, clusterGroupUpgrade, managedPoliciesPresent)
+				err := r.buildRemediationPlan(ctx, clusterGroupUpgrade, managedPoliciesPresent)
```

Comment:
```suggestion
				err = r.buildRemediationPlan(ctx, clusterGroupUpgrade, managedPoliciesPresent)
```
err is already defined in the outer scope, can it be reused here?

Reply:
Yes, thank you :+1: